client/src/components/EmailSignup.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div class='section form-wrapper columns'>
+    <div class='header-section column is-half is-centered'>
+      <div class='h1-header-large centered-text'>
+        {{ MESSAGES.SUBSCRIBE_HEADER }}
+      </div>
+      <div class='h2-header-large centered-text mt-4'>
+        {{ MESSAGES.SUBSCRIBE_SUBHEADER }}
+      </div>
+    </div>
+    <div id='hubspotForm' v-once></div>
+  </div>
+</template>
+
+<script lang='ts'>
+import { defineComponent } from 'vue';
+import { LandingPageMessages } from '../assets/messages/landing';
+
+/**
+ * Section containing form to sign up for LeadOut data updates.
+ *
+ * This wraps a hubspot form which is configured and styled in the hubspot editor.
+ */
+export default defineComponent({
+  name: 'EmailSignup',
+  data() {
+    return {
+      MESSAGES: LandingPageMessages,
+    };
+  },
+  mounted() {
+    // Create script tag which contains logic to attach embedded hubspot form to a div rendered in
+    // the template. This is necessary since vue does not allow script tags in the template.
+    const script = document.createElement('script');
+    script.src = '//js.hsforms.net/forms/v2.js';
+    script.type = 'text/javascript';
+    const formScript = document.createElement('script');
+    document.body.appendChild(script);
+    document.body.appendChild(formScript);
+
+    script.addEventListener('load', () => {
+      if (window.hbspt) {
+        // LeadOut Website Subscribe Form.
+        window.hbspt.forms.create({
+          region: 'na1',
+          portalId: '7810253',
```

Comment:
Do any of these need to be kept secret via ENV?

Reply:
Not that I'm aware of - basically hubspot says copy / paste this to embed in your app, so I think they are all just related to the form, i.e. don't think that they reveal any private info about BCs account. I can double check with Dunrie though.

Reply:
Checked with Dunrie and she isn't sure, but doesn't seem like a concern. Resolving this.